README.md

```diff
@@ -1,41 +1,79 @@
+## Summary
+
+The repository contains instructions to replicate and build upon a classifier
+trained to detect an NF1 inactivation signature in glioblastoma gene expression
+data. We leverage publicly available data from the Cancer Genome Atlas (TCGA) to
+ tune a logistic classifier with an elastic net penalty using stochastic
+gradient descent. NF1 is a tumor suppressor that regulates RAS (a well
+characterized oncogene). When NF1 is inactivated, RAS signaling continues
+unabated leading to uncontrolled cell growth. Patients with neurofibromatosis
+type I (caused by heterozygous germline mutation of NF1) have a predisposition
+for multiple tumor types including optic gliomas, pheochromocytomas, and
+malignant peripheral nerve sheath tumors. Furthermore, NF1 is one of the most
+commonly mutated genes in glioblastoma. NF1 can be inactivated genetically or by
+other mechanisms including microRNAs or targeted degradation by the proteosome
+([McGillicuddy et al. 2009](http://www.ncbi.nlm.nih.gov/pubmed/19573811)).
+Therefore, detecting inactivation solely by sequencing the NF1 gene can result
+in false negatives. Because we have previously identified compounds that are
+synthetically lethal in NF1 inactivated cells
+([Wood et al. 2011](http://www.ncbi.nlm.nih.gov/pubmed/21697395)),
+the ability to detect patients with NF1 inactivation signatures could inform
+treatment decisions.
+
+## Code
+
+* A step-by-step guide of all analyses for the NF1 inactivation project is
+provided in `ANALYSIS.sh`
+* All required R packages are installed by `INSTALL.R`.
+* `scripts/` - all the main functions and pipeline scripts are provided
+  * |- `util/` - several helper functions including:
+    * |- `cancer_cv.py` - cross validation logic to ensure equal partitioning
+    * |- `get_y_class.py` - determine mutation status for genes in `X` matrix
+    * |- `transform.r` - transforming RNAseq distributed data into microarray
+  * |- `viz/` - using ggplot to vizualize classifier performance
+
+## Contact
 
-###################
-# CONTACT
-###################
-Please report all bugs and direct analysis questions to: `gregway@mail.med.upenn.edu`
+Please report all bugs and direct analysis questions to:
```

Comment:
I wrote this before I knew about the awesome power of github issues. I will revise in this pull request, thanks!